recipes/ogdf/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ogdf" %}
+{% set version = "2020.02" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ogdf/ogdf/archive/catalpa-202002.tar.gz
+  sha256: 32f3f943675e55d1a23e62293d016dc1644b1176271d60f41788ade60c2d3fc5
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_subpackage('ogdf', max_pin='2020.02') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.1
+    - make  # [not win]
+    - ninja  # [win]
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libOGDF.so  # [linux]
+    - test -f $PREFIX/lib/libOGDF.dylib  # [osx]
+    - if not exist %LIBRARY_LIB%\\OGDF.lib exit 1  # [win]
+
+about:
+  home: http://www.ogdf.net/
+  license: GPL-2.0-only OR GPL-3.0-only
+  summary: OGDF is a self-contained C++ class library for the automatic layout of diagrams
+  
+  description: |
+    OGDF is a self-contained C++ class library for the automatic layout of diagrams.
+  doc_url: http://www.ogdf.net/doku.php
+  dev_url: https://github.com/ogdf/ogdf
+
+  license_file:
```

Comment:
Any idea how to specify the folder of the recipe for those files since they are not in the source repository?

Reply:
Usually conda-build just finds them. 

However, I see a few options.

1. move the files up a dir so they are just out in the recipe
2. use `{{ os.environ["RECIPE_DIR"] }}` in jinja2

Reply:
Newer `conda-build`s should just work with relative paths from either the source or recipe, I think.
E.g., `license_file: relative/path/to/license` should work if `relative/path/to/license` is in the source archives or just under the recipe path.

Reply:
I will include the licenses in the fork.

Reply:
Sorry I got confused between ogdf and tmap. I won't patch ogdf so I will use `{{ os.environ["RECIPE_DIR"] }}`.

Reply:
xref with https://github.com/conda-forge/staged-recipes/pull/12498/commits/5ce0a55d15d4288d87255d86b30e082102454ec9

Reply:
Unless I am doing something wrong `{{ os.environ["RECIPE_DIR"] }}` does not work. I am not sure how to move the license files up.

Reply:
Error: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=203261&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=504